src/Illuminate/Contracts/Logging/Log.php

```diff
@@ -95,4 +95,11 @@ public function useFiles($path, $level = 'debug');
      * @return void
      */
     public function useDailyFiles($path, $days = 0, $level = 'debug');
+
+    /**
+     * Register a null log handler (disable logging)
+     *
+     * @return void
+     */
+    public function useNull();
```

Comment:
You cannot add this method to the contract. It'll break existing code such as Bugsnag.